assets/queries/cloudFormation/BucketPolicyEnabled/test/positive.yaml

```diff
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: A sample template
+Resources:
+  mybucketVulnerable:
+    Type: "AWS::S3::Bucket"
+    DeletionPolicy: Retain
+    Properties:
+      ReplicationConfiguration:
+        Role:
+          "Fn::GetAtt":
```

Comment:
Linter is complaining about this, Try removing the surrounding double quotes: `Fn::GetAtt`